node/src/utils/assertions.js

```diff
@@ -0,0 +1,25 @@
+const { options } = require('../config/constants');
+
+const { API_KEY, API_SECRET, ORGANIZATION_ID, PROJECT_ID, ENVIRONMENT_NAME } = options;
+
+const assertDeploymentStatus = status => {
```

Comment:
This is business logic, so I wouldn't put it in a general `util`.
IMO `assertDeploymentStatus` is part of `deploy-utils`,
`assertRequiredOptions` is part of whatever is managing the configuration.

Reply:
3608b42